core/src/main/java/org/frankframework/parameters/Parameter.java

```diff
@@ -198,10 +198,10 @@ public void configure() throws ConfigurationException {
 		if (StringUtils.isNotEmpty(getSessionKey()) && StringUtils.isNotEmpty(getSessionKeyXPath())) {
 			throw new ConfigurationException("Parameter ["+getName()+"] cannot have both sessionKey and sessionKeyXPath specified");
 		}
+		if (paramList!=null) {
+			paramList.configure();
+		}
```

Comment:
Hebben we parameters die parameters hebben? Tjeemig, dat had ik me nog niet eerder gerealiseerd!

Reply:
We gebruiken ook bijna nooit parameters voor onze parameters. Zal wel alleen gebruikt worden voor het parameteriseren van de XPath expressions.

Had je tests toegevoegd voor parameters met parameters? (In de XPath expression of ergens anders).
Zover was ik nog niet gekomen in het bekijken van de ParameterTest en toen zag ik dat het al gemerged was 😬